src/resmom/start_exec.c

```diff
@@ -648,6 +648,14 @@ open_std_out_err(job *pjob)
 		return -1;
 	}
 
+	if (!direct_write_possible && direct_write_requested(pjob)) {
```

Comment:
This will get changed by the following line.
Ln: 5117: `if (is_direct_write(pjob, which, path, &direct_write_possible)) {`
The value will be zero if direct_write is not possible for either of the stdout or stderr files.